code/localization/localize.cpp

```diff
@@ -190,20 +190,22 @@ void parse_stringstbl_quick(const char *filename)
 
 				if (!Unicode_text_mode) {
 					required_string("+Special Character Index:");
-					stuff_ubyte(&language.special_char_indexes[0]);
+					ubyte temp;
+					stuff_ubyte(&temp);
+					language.special_char_indexes.push_back(temp);
 					for (i = 1; i < LCL_MAX_FONTS; ++i) {
 						// default to "none"/0 except for font03 which defaults to 176
 						// NOTE: fonts.tbl may override these values
 						if (i == font::FONT3) {
-							language.special_char_indexes[i] = 176;
+							language.special_char_indexes.push_back(176);
 						} else {
-							language.special_char_indexes[i] = 0;
+							language.special_char_indexes.push_back(0);
 						}
 					}
 				} else {
 					// Set all indices to valid values
-					for (i = 0; i < LCL_MAX_FONTS; ++i) {
-						language.special_char_indexes[i] = 0;
+					for (i = 1; i < LCL_MAX_FONTS; ++i) {
+						language.special_char_indexes.push_back(0);
```

Comment:
Same question re `push_back` vs. `[]` index.

Reply:
Here the languages are being defined for the first time, and the vectors are being built, so attempting to access via `[] index` would result in a crash.